lib/tpl/dokuwiki/css/_edit.css

```diff
@@ -123,6 +123,8 @@ div.picker button.toolbutton {
 .dokuwiki .secedit {
     float: right;
     margin-top: -1.4em;
+    margin-bottom: 1.4em;
+    clear: both;
```

Comment:
If you have TOC on the right hand of the page, `clear:both` will push all edit buttons on the left of TOC to be under TOC, which is not acceptable.

![demo](https://user-images.githubusercontent.com/331403/72687621-c077ba00-3acd-11ea-9c81-e775e5bd1eb8.png)